mod_app/admin/film_admin.py

```diff
@@ -28,11 +43,36 @@ class TRInline(admin.TabularInline):
 
 @admin.register(Film)
 class FilmAdmin(admin.ModelAdmin):
-    inlines = [FilmAnalysisInline, TRInline]
+    class Media:
+        css = {
+            "all": ("admin/css/custom.css",),
+        }
+
+    autocomplete_fields = ["genre"]
+    search_fields = ["title", "alt_titles"]
+    formfield_overrides = {
+        models.TextField: {"widget": CKEditorWidget},
```

Comment:
If it's not too much effort, I'd suggest paring down the toolbar on some of the film admin text fields like cast, credits (I doubt they'll need emojis 😸)

Reply:
That might be a bit tricky since I'm overriding the field type and not defining each one individually to try keep it DRY